app/resources/api/v2/sample_metadata_resource.rb

```diff
@@ -10,6 +10,8 @@ class SampleMetadataResource < BaseResource
       attribute :cohort
       attribute :sample_description
       attribute :donor_id
+      attribute :concentration
```

Comment:
I'm not comfortable with concentration and volume being in sample metadata.
Samples are the abstract sample details, not about physical instances of samples in labwares. Concentration and volume are associated with physical wells and tubes in Sequencescape.

Reply:
Discovered the columns are in manifests and get uploaded in specialised fields to both sample_metadata and as a qc result. See app/sequencescape_excel/sequencescape_excel/specialised_field/volume.rb